src/plugins/auto/utils/config.hpp

```diff
@@ -27,8 +27,69 @@ struct PluginConfig {
                 _logLevel("LOG_NONE") {
         adjustKeyMapValues();
     }
+    std::vector<std::string> supportedConfigKeys(const std::string& pluginName = "AUTO") const {
+        std::vector<std::string> supported_configKeys = []() -> decltype(PerfHintsConfig::SupportedKeys()) {
+            auto res = PerfHintsConfig::SupportedKeys();
+            res.push_back(ov::device::priorities.name());
+            res.push_back(ov::enable_profiling.name());
+            res.push_back(PluginConfigParams::KEY_EXCLUSIVE_ASYNC_REQUESTS);
+            res.push_back(ov::hint::model_priority.name());
+            res.push_back(ov::hint::allow_auto_batching.name());
```

Comment:
probably allow_auto_batch is not needed anymore?

Reply:
will be fix at next PR related to AUTO/MULTI update.